src/components/atoms/UserAvatar/UserAvatar.scss

```diff
@@ -20,6 +20,19 @@
     height: 8px;
     width: 8px;
     z-index: z(user-avatar-status-dot);
+
+    &--online {
+      background-color: $green;
+    }
+    &--busy {
+      background-color: $ripple-color;
+    }
+    &--offline {
+      background-color: $red;
+    }
```

Comment:
The logic of a user being `online` status is defined in our `hooks/users` `useRecentWorldUsers` hook. If we try to set it to `status`, we will have 2 sources of truth, which is incorrect. That's why, this state should not be stored there. 
Offline status is the default status of the user and that's why we should't expect it to explicitly come from `status` profile field

Reply:
> The logic of a user being online status is defined in our hooks/users useRecentWorldUsers hook. If we try to set it to status, we will have 2 sources of truth, which is incorrect. That's why, this state should not be stored there.

@mike-lvov This seems unrelated to the styles changes here?

Reply:
https://github.com/sparkletown/sparkle/pull/1384/files#diff-edfb3f24596e6fe740f3de347baec9272f305ca2717d22bca5ac4fc5e2eb3b04R37